packages/documentation/src/stories/foundation/list/list-reset.sample.scss

```
@@ -0,0 +1,5 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+ul.my-list {
+  @include post.list-reset();
+}
```

Including the whole styles library is not necessary for using the reset and the mixin name is `reset-list`.

```suggestion
@use '@swisspost/design-system-styles/mixins/utilities' as post;

ul.my-list {
  @include post.reset-list();
}
```

---

Fixed the mixin name.

The core file only contains `variables`, `mixins` and `functions`. This means, that nothing is added to the CSS output, except what is actually used. With this in mind, it does not make sense, to add more specific import statements imo.
What do you think?